Framework/Crystal/src/IntegratePeakTimeSlices.cpp

```diff
@@ -427,7 +422,7 @@ void IntegratePeakTimeSlices::exec() {
           if (m_R0 > 0)
             Radius = m_R0;
 
-          int NN = m_NeighborIDs[1];
+          const int NN = m_NeighborIDs[1];
```

Comment:
Can we change this to `neighbourId` as well to bring it inline with the change below? Or if we need to have separate variables can we use a name which explains why we need two `neighbourId`s?

Reply:
I'll remove both temporary variables to solve the issue. That seems the most straightforward thing to do as I don't want to dive into the code to invent more spot-on names.